packages/berlin/src/pages/Comments.tsx

```diff
@@ -82,7 +85,6 @@ function Comments() {
     queryKey: ['option', optionId, 'comments'],
     queryFn: () => fetchComments({ optionId: optionId || '' }),
     enabled: !!optionId,
-    // refetchInterval: 5000, // Poll every 5 seconds
```

Comment:
Are we removing polling?

Reply:
it was commented, just thought to remove the comment